### PR TITLE
ROX-3791: Fix grammar for ADD default policy

### PR DESCRIPTION
**Pull request by srcporter**
## Description

Single character added to correct grammar for the description of the policy "ADD Command used instead of COPY"

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Not tested - changes only a string value for description

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Review by vjwilson — APPROVED** [review]

**Comment by clickboo** [comment]
This will need a migration for existing customers who upgrade to get the fix.

**Comment by vjwilson** [comment]
> This will need a migration for existing customers who upgrade to get the fix.

@srcporter Does it matter if existing customers see the new wording?

File: migrator/migrations/n_39_to_n_40_postgres_policies/legacy/files/add_instruction.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "80267b36-2182-4fb3-8b53-e80c031f4ad8",
   "name": "ADD Command used instead of COPY",
-  "description": "Alert on deployments using a ADD command",
+  "description": "Alert on deployments using an ADD command",
```
Comment (theencee) [inline migrator/migrations/n_39_to_n_40_postgres_policies/legacy/files/add_instruction.json:4]: Please don't change this file. Nothing in existing migration files should be modified

**Comment by clickboo** [comment]
> > This will need a migration for existing customers who upgrade to get the fix.
> 
> @srcporter Does it matter if existing customers see the new wording?

While I understand it might seem overkill for a single letter change, I would not advise diverging policies in a fresh install vs upgrade to same tag path. Also +1 to what @theencee - we must not change any migration. A migration is frozen once released.

**Comment by theencee** [comment]
@clickboo There's a ticket (https://issues.redhat.com/browse/ROX-16950) for adding some more exclusions in 4.1. While not ideal, I think I could probably just combine a migration for this onto that one

**Comment by theencee** [comment]
Reminder: Merge this PR once https://github.com/stackrox/stackrox/pull/6036 is merged. 

If it'll miss RC0 cutoff for 4.1, be sure to add it to RC1

**Pull request merged**